Zend/zend_operators.h

```diff
@@ -490,6 +490,20 @@ overflow: ZEND_ATTRIBUTE_COLD_LABEL
 	return;
 overflow: ZEND_ATTRIBUTE_COLD_LABEL
 	ZVAL_DOUBLE(op1, (double)ZEND_LONG_MAX + 1.0);
+#elif defined(HAVE_ASM_GOTO) && defined(__aarch64__)
+	register long tmp asm("x5");
```

Comment:
Any reason to hardcode this register?

Reply:
without `tmp` being fixed to a given register, gcc decides to use a register already in use, and that causes wrong code.

Reply:
You should reserve a register, putting it in the "Clobbers" list. See https://gcc.gnu.org/onlinedocs/gcc/Extended-Asm.html

Reply:
Thank you @dstogov for your guidance: I declared the temporary register x5 in the clobber list and the compiler would not use that register.  `make test` passes on the updated patch.